test/fixtures/users.yml

```diff
@@ -7,6 +7,7 @@
 one:
   first_name: Testy McTestface
   email: test@test.com
+  encrypted_password: <%= Devise::Encryptor.digest(User, 'password') %>
 #
```

Comment:
Theres a built-in Encryptor!? 😱

Reply:
Devise does a lot of stuff.. maybe too much. There has been a couple of times today when I regretted not just rolling our own authentication.